include/internal/provider.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef OSSL_INTERNAL_PROVIDER_H
+# define OSSL_INTERNAL_PROVIDER_H
+
+# include <openssl/core.h>
+# include "internal/dso.h"
+
+# ifdef __cplusplus
+extern "C" {
+# endif
```

Comment:
Internal header file so this isn't needed.

Reply:
Okie